packages/nimble-components/src/table-column/number-text/tests/table-column-number-text-matrix.stories.ts

```diff
@@ -42,18 +42,26 @@ const alignmentStates: [string, string | undefined][] = Object.entries(
 ).map(([key, value]) => [pascalCase(key), value]);
 type AlignmentState = (typeof alignmentStates)[number];
 
+const placeholderStates: [string, string | undefined][] = [
+    ['With Placeholder', 'Custom placeholder'],
+    ['', undefined]
+];
```

Comment:
Now that I've made this suggestion several times, I think it might be worthwhile to move this to `utilities/tests/states.ts`.

```suggestion
const placeholderStates = [
    ['With Placeholder', 'Custom placeholder'],
    ['', undefined]
] as const;
```

Reply:
Moved to `utilities/tests/states.ts`.